src/main/java/tests/OperationStrategyTest.java

```diff
@@ -0,0 +1,25 @@
+package tests;
+
+import core.basesyntax.*;
```

Comment:
It's better to import classes explicitly rather than using wildcard imports. This helps in understanding which exact classes are being used in the test.